.github/workflows/test_iree.yml

```diff
@@ -115,3 +115,78 @@ jobs:
             --retries 2 \
             --retry-delay 5 \
             --durations=0 \
+            --config-files=iree_tests/configs/config_pytorch_models_cpu_llvm_task.json
+          pytest iree_tests/pytorch/models/sdxl-scheduled-unet-3-tank \
```

Comment:
nit: I'd split this into its own job step

Reply:
I like being able to click through each part of a job and see the time breakdown for each:
![image](https://github.com/nod-ai/SHARK-TestSuite/assets/4010439/fc7c403c-7a63-4d58-b157-a20d1bb88a85)

This a case where we'd be optimizing for the reader of the CI logs and not the reader/writer of the workflow file

Reply:
Ah I see, ok cool